src/applications/rated-disabilities/components/NoCombinedRating.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export default function NoCombinedRating() {
+  return (
+    <va-alert status="info" uswds>
```

Comment:
if we are adding a new va-alert to rated disabilities that has uswds enabled then we will want to update the ticket [here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76124) and spreadsheet [here](https://docs.google.com/spreadsheets/d/1ajKjN6dr-2PzpgEbfX1OqxAupvbqoHwqh7ml1m5a1Fo/edit?usp=sharing) to make sure we account for this work.

Reply:
Added to spreadsheet